data/content/agent_attributes.yaml

```
@@ -115,6 +115,13 @@ attributes:
   desc: |
       Path to where mirrors of git repositories are stored. Refer to [Git mirrors](/docs/agent/v3#promoted-experiments-git-mirrors) for more information on this feature. Introduced in [v3.47.0](https://github.com/buildkite/agent/releases/tag/v3.47.0).
       _Example:_ `/tmp/buildkite-git-mirrors`
+name: git-mirrors-skip-update
```

Missing dash for YAML syntax:

```suggestion
- name: git-mirrors-skip-update
```